### ISSUE TITLE
[TASK][TRIVIAL] Fix typo in README.md file

### ISSUE THREAD

**Issue by pan3793** [apache/kyuubi#5432]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before creating

- [X] I have searched in the [task list](https://github.com/orgs/apache/projects/296) and found no similar tasks.

### Mentor

- [ ] I have sufficient knowledge and experience of this task, and I volunteer to be the mentor of this task to guide contributors to complete the task.

### Skill requirements

- Basically English skills

### Background and Goals

Fix typo in README.md file

### Implementation steps

Fix typo in README.md file

**Comment by omimakhare** [comment]
I have created a PR here. https://github.com/apache/kyuubi/pull/5431 you can assign it  to me.
